### ISSUE TITLE
nosuchalgorithmexception when installing native-image on github actions.

### ISSUE THREAD

**Issue by Wqrld** [oracle/graal#3173]
**Describe the issue**
installing native image using gu on the dev builds results in a error:

```
Run graalvm-ce-java11-21.1.0-dev/bin/gu install native-image
  graalvm-ce-java11-21.1.0-dev/bin/gu install native-image
  shell: /bin/bash -e {0}
  env:
    JAVA_HOME_11.0.10_x64: /opt/hostedtoolcache/jdk/11.0.10/x64
    JAVA_HOME: /opt/hostedtoolcache/jdk/11.0.10/x64
    JAVA_HOME_11_0_10_X64: /opt/hostedtoolcache/jdk/11.0.10/x64
Downloading: Component catalog from raw.githubusercontent.com
Error: Error downloading component catalog from https://raw.githubusercontent.com/graalvm/graalvm-ce-dev-builds/03c14b902825b849b36eb06680d27470d1664429/catalog.jdk_11.properties: java.security.NoSuchAlgorithmException: class configured for SSLContext (provider: SunJSSE) cannot be found.. 
Please check your connection and proxy settings. If your machine is behind a proxy, environment variables (http_proxy, https_proxy, ...) must be set appropriately.
Error: Process completed with exit code 3.
|
```

**Steps to reproduce the issue**
download, extract graal and gu install native-image

**Describe GraalVM and your environment:**
Graal: https://github.com/graalvm/graalvm-ce-dev-builds/releases/download/21.1.0-dev-20210126_1929/graalvm-ce-java11-linux-amd64-dev.tar.gz
Env: github runner ubuntu 18.04 LTS

**Comment by Wqrld** [comment]
Friend of me said he was able to reproduce this locally. I haven't had the change to try it out locally.

**Comment by warrenc5** [comment]
Yes I'm also getting this. I tried installing JSE manually - no success.
Workaround is to download and install manually gu install -L 
wget -P ~/Downloads/ https://github.com/graalvm/graalvm-ce-dev-builds/releases/download/21.1.0-dev-20210126_1929/native-image-installable-svm-java11-linux-amd64-dev.jar

sudo /usr/local/java/graalvm/bin/gu install -L ~/Downloads/native-image-installable-svm-java11-linux-amd64-dev.jar

**Comment by kkriske** [comment]
Could this be the same thing as #3157? `gu` is in itself a native-image if I'm not mistaken, so seems like it's also missing the security services?

**Comment by mcraj017** [comment]
@Wqrld Thank you for reporting the issue
we will check it out and come back to you

**Comment by MartinBalin** [comment]
As a workaround run it in JVM mode, if this is the source of problems. `--jvm`

**Comment by sdedic** [comment]
@Wqrld just to confirm: did the error always involved the `nosuchalgorithmexception`  such as in #3157 ?

**Comment by Wqrld** [comment]
> @Wqrld just to confirm: did the error always involved the `nosuchalgorithmexception` such as in #3157 ?

It has always been this exact error. #3157 is also a nosuchalgorithmexception but seems to be unrelated everywhere else

**Comment by sdedic** [comment]
good. GU itself does not mess with JCE in any way. If the error is related to #3157, then running `gu` with `--jvm` parameter should fix that (does not run as native image but in JVM mode). I'd close this issue then (fix should be already in for 21.1.0)  - can you confirm that `--jvm` works as a workaround ?

**Comment by Wqrld** [comment]
> good. GU itself does not mess with JCE in any way. If the error is related to #3157, then running `gu` with `--jvm` parameter should fix that (does not run as native image but in JVM mode). I'd close this issue then (fix should be already in for 21.1.0) - can you confirm that `--jvm` works as a workaround ?

can confirm that --jvm works. Compiles like a charm

**Comment by sdedic** [comment]
Good, closed: for fix details, refer to #3157.
